src/modules/displayTask.js

```diff
@@ -0,0 +1,59 @@
+import check from './check.js';
+
+const tasksList = document.querySelector('.tasks-list');
+
+// const todos = JSON.parse(localStorage.getItem('todo-list'));
+
+// function deleteFunction(id) {
+//   console.log(todos);
+//   console.log(id);
+//   // todos = todos.splice(id, 1);
+//   // localStorage.setItem('todo-list', JSON.stringify(todos));
+//   // console.log(todos);
+// }
```

Comment:
- [ ] Please do not push commented codes to your repo. Comments render your work messy.